.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
Do we know if this new system groups issue created via templates in such a way where we wouldn't need "feature request" or "bug" labels?

Reply:
I think it is intended to auto-assign "bug" and "new feature" labels.